src/Autocomplete/Autocomplete.tsx

```diff
@@ -98,3 +100,38 @@ function Option(
     </Box>
   );
 }
+
+// renderer for a tooltip icon
+function OptionWithTooltip(
+  props: React.HTMLAttributes<HTMLLIElement>,
+  option: KeyValueOption | string
+) {
+  if (typeof option === "object") {
+    return (
+      <Box
+        {...props}
+        component="li"
+        sx={{
+          justifyContent: "space-between !important"
```

Comment:
Whats the need for the important here? What are we have to force override?

Reply:
![ScreenShot Tool -20231115091217](https://github.com/IPG-Automotive-UK/react-ui/assets/56511816/a7d82bfd-d2f8-4e2c-9b31-ebd56cfba7fe)
 I am forcing style override to achieve this. By default, the li is "justify-content: "flex-start", this was not working by just adding "justifyContent": "space-between", so I am forcing it.